### PR TITLE
Implement RemoveShield for easier management of stripping demoshields / razorbacks

### PR DESCRIPTION
**Pull request by shadow93**
easier code management, for efficiency's sake.

### CODE REVIEW

**Comment by naydef** [comment]
Why not just remove shield[client] array and use FindEntityByClassname to find the shield?

File: addons/sourcemod/scripting/freak_fortress_2.sp [hunk 1]
```diff
@@ -8251,6 +8241,16 @@ public Action:Timer_UseBossCharge(Handle:timer, Handle:data)
 	return Plugin_Continue;
 }
 
+public StripShield(client, attacker, Float:position[3])
```
Comment (50Wliu) [inline addons/sourcemod/scripting/freak_fortress_2.sp]: stock instead of public

File: addons/sourcemod/scripting/freak_fortress_2.sp [hunk 2]
```diff
@@ -12,6 +12,10 @@ Plugin thread on AlliedMods: http://forums.alliedmods.net/showthread.php?t=18210
 Updated by Otokiru, Powerlord, and RavensBro after Rainbolt Dash got sucked into DOTA2
 
 Updated by Wliu, Chris, Lawd, and Carge after Powerlord quit FF2
+
+Some code courtesy of Koishi/Shadow's FF2 fork.
+
+Also, for some reason, a certain individual seems to have a deep mancrush on one of our contributors. Please stop, it's getting pretty annoying.
```
Comment (50Wliu) [inline addons/sourcemod/scripting/freak_fortress_2.sp]: no

File: addons/sourcemod/scripting/freak_fortress_2.sp [hunk 3]
```diff
@@ -8244,6 +8244,7 @@ stock RemoveShield(client, attacker, Float:position[3])
 	EmitSoundToClient(client, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
 	EmitSoundToClient(attacker, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
 	EmitSoundToClient(attacker, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
+	TF2_AddCondition(client, TFCond_Bonked, 0.1); // is this even needed?
```
Comment (Deathreus) [inline addons/sourcemod/scripting/freak_fortress_2.sp]: It is, the split second it get's applied before being hit makes it show the MISS!

File: addons/sourcemod/scripting/freak_fortress_2.sp [hunk 4]
```diff
@@ -8251,6 +8237,17 @@ public Action:Timer_UseBossCharge(Handle:timer, Handle:data)
...
 }
 
+stock RemoveShield(client, attacker, Float:position[3])
+{
+	TF2_RemoveWearable(client, shield[client]);
+	EmitSoundToClient(client, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
+	EmitSoundToClient(client, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
+	EmitSoundToClient(attacker, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
+	EmitSoundToClient(attacker, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
+	TF2_AddCondition(client, TFCond_Bonked, 0.1); // is this even needed?
```
Comment (50Wliu) [inline addons/sourcemod/scripting/freak_fortress_2.sp]: Get rid of the comment and I'll merge.
(or alternatively update it to indicate that this line is there to show the "MISS!" line)

Comment (shadow93) [inline addons/sourcemod/scripting/freak_fortress_2.sp]: done.

**Pull request merged**